src/OrchardCore.Modules/OrchardCore.Menu/Migrations/menu.recipe.json

```diff
@@ -98,6 +132,34 @@
               "Description": "Provides a menu link behavior for your content item."
             }
           }
+        },
+        {
+          "Name": "ContentPickerMenuItemPart",
+          "Settings": {
+            "ContentPartSettings": {
+              "Attachable": true,
+              "DisplayName": "ContentPickerMenuItem",
+              "Description": "Provides a menu link behavior from ContentPicker for your content item."
+            }
+          },
+          "ContentPartFieldDefinitionRecords": [
+            {
+              "FieldName": "ContentPickerField",
+              "Name": "LinkToContentItem",
+              "Settings": {
+                "ContentPartFieldSettings": {
+                  "DisplayName": "Link To Content Item",
+                  "Position": "0"
+                },
+                "ContentPickerFieldSettings": {
+                  "Required": true,
+                  "Multiple": false,
+                  "DisplayedContentTypes": []
```

Comment:
We need a second migration here, just for the `LinkToContentItem` field, to add it to existing sites.

Suggest implementing a second migration
`UpdateFrom1()` which returns 2, and in the `Create()` migration, also return 2, which will shortcut the second migration on existing sites

Reply:
I understand the migration for exsiting sites,but i am sorry to need your more explanation about here,I have accomplished this as my new commit, please give me a help.